src/components/SendTransaction.vue

```diff
@@ -0,0 +1,49 @@
+<script setup>
+import { useWallet } from 'solana-wallets-vue'
+import { Connection, clusterApiUrl, Keypair, SystemProgram, Transaction } from '@solana/web3.js';
+
+const { publicKey, connected, sendTransaction } = useWallet();
+
+const onClick = async () => {
+        if (!publicKey) {
+            console.log('error', `Send Transaction: Wallet not connected!`);
+            return;
+        }
+
+        let signature = '';
+        try {
+            const transaction = new Transaction().add(
+                SystemProgram.transfer({
+                    fromPubkey: publicKey.value,
+                    toPubkey: Keypair.generate().publicKey,
+                    lamports: 1,
+                })

```

Comment:
It's nice to use a simple transaction by default but I'd maybe add a comment explaining what this does because we're ultimately making the user lose money. Imagine they switch `lamports` to something much bigger than one and use the `mainnet` cluster locally to test things. Not a very nice onboarding experience haha.

Reply:
Good point, added a comment! Though I believe the wallet extension also prompts the user with a confirmation popup before the transaction goes through (at least with my phantom wallet it does)